### ISSUE TITLE
Narrator cannot read the contents of the Expander header in scan mode

### ISSUE THREAD

**Issue by Richasy** [microsoft/microsoft-ui-xaml#5820]
**Describe the bug**
We are using `Expander` as a container for setting items, and we have placed two lines of text in `Expander.Header`. We hope that the narrator can read the contents of the Expander head in scan mode, but it cannot do so at the moment.

**Steps to reproduce the bug**
1. Open narrator (shortcut is `Ctrl`+`Win`+`Enter`).
2. Open scan mode (shortcut is `Caps Lock`+`Space`).
3. Create an Expander containing two lines of text, such as code below.
4. Use keyboard to focus expander, press down arrow keyboard button.

```xml
<muxc:Expander HorizontalAlignment="Stretch">
    <muxc:Expander.Header>
        <StackPanel>
            <TextBlock Text="dasdasdasdsadsa" />
            <TextBlock Text="dasds" />
        </StackPanel>
    </muxc:Expander.Header>
</muxc:Expander>
```

**Expected behavior**
The narrator will read the contents of the Expander header.

**Actual behavior**
Narrator jumps directly to the next focusable element.

**Version Info**
2.6.2

NuGet package version: 
[Microsoft.UI.Xaml 2.6.2]

| Windows version                    | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (22000)              |  Yes   |
| May 2021 Update (19043)            |    |
| October 2020 Update (19042)        |    |
| May 2020 Update (19041)            |    |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            | Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Comment by StephenLPeters** [comment]
This works with one text block because narrator is able to interpret the contents of the content present and read the string, but when the child of the content presenter is a stack panel, it doens't know what to do.  I think that narrator is then unable to scan this content because the content presenter sets its accessibility view to raw in the template here: https://github.com/microsoft/microsoft-ui-xaml/blob/c94b88625232fc588c86e0da5ffc090b7a599834/dev/Expander/Expander_themeresources.xaml#L624

I think if we removed that setting the experience would be better. @tashatitova do you know if there is a reason this is raw?

**Comment by greedyAI** [comment]
Hi @StephenLPeters I tried the following things and none of them worked:

- Removing the ContentPresenter's AutomationProperties.AccessibilityView="Raw" property in ExpanderHeaderUpStyle and ExpanderHeaderDownStyle.
- Setting the ContentPresenter's AutomationProperties.AccessibilityView property to Content and Control in ExpanderHeaderUpStyle and ExpanderHeaderDownStyle.
- Changing the ContentPresenter to a ContentControl in ExpanderHeaderUpStyle and ExpanderHeaderDownStyle.
- Setting AutomationProperties.AccessibilityView to Raw, Content, and Control in DefaultExpanderStyleCustom's ToggleButton
- Setting my own Expander.Header's subcontrols' AutomationProperties.AccessibilityView properties to Raw, Content, and Control.

Is this an issue in the code behind for WinUI that we can't fix via a custom style? Please let me know if there are any other workarounds that I can try! Thanks!
